lightning-invoice/src/payment.rs

```diff
@@ -45,7 +48,7 @@
 //! # use lightning::util::logger::{Logger, Record};
 //! # use lightning::util::ser::{Writeable, Writer};
 //! # use lightning_invoice::Invoice;
-//! # use lightning_invoice::payment::{InvoicePayer, Payer, Retry, Router};
+//! # use lightning_invoice::payment::{InvoicePayer, Payer, Retry, Router, InFlightHtlcs};
```

Comment:
nit: order lexicographically

Reply:
Addressed in c5206b3bf646b61db957ef7dd80b79538114181d